src/vs/platform/terminal/node/terminalProcess.ts

```diff
@@ -220,6 +239,7 @@ export class TerminalProcess extends Disposable implements ITerminalChildProcess
 		this._onProcessExit.dispose();
 		this._onProcessReady.dispose();
 		this._onProcessTitleChanged.dispose();
+		this._onShellTypeChanged.dispose();
```

Comment:
We don't actually need to dispose any of these events because they're `this._register`ed